elements/rh-footer/rh-global-footer.ts

```diff
@@ -3,7 +3,7 @@ import type { ColorPalette } from '@patternfly/pfe-core';
 import { LitElement, html } from 'lit';
```

Comment:
Missed import link for ColorPalette RHDS lib instead of pfe-core

```
import type { ColorPalette } from '../../lib/context/color.js';
```

Reply:
done